### DISCUSSION TITLE
How can i make a button stay at the end of a resizable div?

### DISCUSSION THREAD

**Discussion by diogoribeirodev** [tailwindlabs/tailwindcss#12669] [Help]
Hi, i'm trying to mimic resizable sidebar by this website raindrop.io ( doesnt use tailwind), i am using the Resizable Panel by @shadcn/ui and i have a combobox at the top of ( that has a dialog with links to other pages( settings/profile...) and a button that will open a modal after clicked), my problem is that i suck at css and i cant make the button follow the div as it get bigger, he does move position but doesnt stay at the end as i want, not only that but the combobox also moves position and i dont want that, i want her to stay where she is and only grow a little bigger! I provided a video of the expected result, and what i have, the screenshots is my code! Can someone help me?
[screen-capture (1).webm](https://github.com/tailwindlabs/tailwindcss/assets/81532541/c8f39df6-1faa-481d-8213-c994385017ad)
[screen-capture.webm](https://github.com/tailwindlabs/tailwindcss/assets/81532541/26937761-bcfd-49d9-bfb2-90c8d8c17a5d)
![image](https://github.com/tailwindlabs/tailwindcss/assets/81532541/d3596eda-ab44-48d1-af04-d9d9463c9e7e)
![image](https://github.com/tailwindlabs/tailwindcss/assets/81532541/7d588eb3-0a04-4e29-bc7e-16adcecee4ab)

**Comment by wongjn** [comment]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) of the rendered HTML, or a project that reproduces the undesired behavior please.

---

As an aside, [please do not upload images of code/data/errors](https://meta.stackoverflow.com/q/285551).

**Reply by wongjn** [reply]
> i cant make one because of the external libraries being used, but i have it deployed at https://linksaver-kappa.vercel.app/dashboard , i can provide the source code if needed( it's going to be an open source project in the future)[^1]

Could you clarify your meaning on how you want "the button follow the div as it get bigger"? Follow in what way? Which button? There could be several UI elements there that could be inferred to be a button. "the combobox also moves position and i dont want that" what position did you want the combobox to be at? How does it move position, on its own or does some user interaction trigger the movement? "i want her to stay where she is and only grow a little bigger!" Who/what are you referring to as "her/she"? Could you define "a little bigger" please?

Also, consider providing a [Tailwind Play](https://play.tailwindcss.com) of the rendered HTML, or a project that reproduces the undesired behavior if the external libraries used are not required to reproduce the undesired behavior.

[^1]: [Comment](https://github.com/tailwindlabs/tailwindcss/discussions/12669#discussioncomment-7967185)

**Comment by diogoribeirodev** [comment]
i cant make one because of the external libraries being used, but i have it deployed at https://linksaver-kappa.vercel.app/dashboard , i can provide the source code if needed( it's going to be an open source project in the future)

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/12669#discussioncomment-7967272 to keep the reply chain together.

**Comment by radumro** [comment]
is this what you want to achieve?
[https://play.tailwindcss.com/V9HecztbBm](https://play.tailwindcss.com/V9HecztbBm)


if you want the user to remain in the middle you an add an extra empty div

[https://play.tailwindcss.com/fI6CxWm25o](https://play.tailwindcss.com/fI6CxWm25o)

ps. you can resize by dragging the bar between code editor  and preview or by resizing the textbox
